bin/build_and_run_jar.sh

```diff
@@ -34,7 +34,6 @@ if [[ $BINARY == "worker" ]]; then
   DEBUG_PORT=5006
```

Comment:
as a sanity check, can you run the docker script (which calls this script) to make sure there are no issues compiling the gradle jar into our images? you can test this via 

.config/gcp/build_and_upload_docker_image.sh api local

which should prompt you to build an image and run it locally (don't need to upload it to GCP)

Reply:
Done. Works!